pkg/authz/distributedtx/activity_test.go

```diff
@@ -96,7 +99,8 @@ func TestCheckKubeResourceError(t *testing.T) {
 	ah := ActivityHandler{KubeClient: trt.toKubeClient()}
 
 	_, err := ah.CheckKubeResource(context.Background(), &KubeReqInput{
-		RequestInfo: &request.RequestInfo{Path: "a_path", Namespace: "ns1"},
+		RequestInfo: &request.RequestInfo{Path: "a_path", Namespace: "ns1", Verb: "get"},
+		RequestURI:  "/a_path",
 		ObjectMeta:  &metav1.ObjectMeta{Name: "object_name"},
 	})
```

Comment:
Is there a test that captures the behavior of passing through the querystring parameters in question?